README.md

```diff
@@ -6,22 +6,22 @@ This is a simple script to do a controlled rolling restart of a multinode elasti
 
 * -m - MASTER - The master node to use for coordinating the update.
 * -n - NODE_FILE - A file containing the list of node hostnames one per line.
-* -s - SCRIPT - Script to run on each node to process the update. 
+* -s - SCRIPT - Script to run on each node to process the update.
 
 ```
 # ./rolling-restart.sh -m master1.example.com:9200 -n ./node-list.txt -s ./node-update.sh
 ```
 
 ### NODE_FILE
 
-The node file is a list of node names (including HTTP port) one per line. These should only be nodes that contain data and the node list should not contain the host you're using as the master node.
+The node file is a list of node names (including HTTP port) one per line. These should only be nodes that contain data and the node list should not contain the host you're using as the master node.  Each line in the node file can contain a comma delimted list of paramaters to pass in to the startup and shutdown scripts.
```

Comment:
I'd do a little rework on this section of the readme and keep the example of a minimal node list

```
node1.example.com:9200
node2.example.com:9200
node3.example.com:9200
```

and then explain how the expanded node list can be used.

Erik simplified this for cases where each node needs the same variables by just exporting them in the calling context and then using them within the update script ... or something along those lines.  I think the expanded node_cfg could still be useful when nodes don't share values.